src/libpsl-native/src/setdate.h

```diff
@@ -17,6 +17,6 @@ typedef struct setDateInfo
     int32_t DST;
 } SetDateInfo;
 
-int32_t SetDate(const SetDateInfo &info);
+int32_t SetDate(const SetDateInfo info);
 
```

Comment:
In my experience, that just means we have no tests, or at least insufficient tests.